shared-libs/message-utils/package-lock.json

```diff
@@ -31,7 +31,21 @@
       }
     },
     "@medic/phone-number": {
-      "version": "file:../phone-number"
+      "version": "file:../phone-number",
```

Comment:
`npm ci` works in both `./share-iibs/message-utils` dir and in the parent `/.cht-core`. Safe to keep?

Reply:
never mind - let's follow best practice and revert.  Fixed in 2ceb94c67 . retested and `npm ci` in both of the dirs still works. thanks thanks!

Reply:
OK - this was a worse idea! When I reverted in https://github.com/medic/cht-core/commit/2ceb94c677bb5476837531e05c5a228bbec51341, while `npm ci` worked, running either api or sentinel with `node server.js` I would get an error (see below).

so re-commiting the changes in 67819fe77  and re-asking "it compiles w/o errors - ok to keep?" ;)


```
➜  sentinel git:(7495-update-dendencies-redux) node server.js  
2022-03-01 16:54:12 INFO: Running server checks…                  
Node Environment Options: 'undefined'                                                 
Node Version: 14.19.0 in development mode                                             
COUCH_URL http://localhost:5984/medic                                                 
COUCH_NODE_NAME nonode@nohost                                                         
Environment variable "COUCH_NODE_NAME" matches server "nonode@nohost"    
CouchDB Version: 2.3.1                                                                
2022-03-01 16:54:12 INFO: Api is ready: {"ready":true,"handler":"medic-api","version":"0.1.0","detail":"All required services are running normally"} 
2022-03-01 16:54:12 ERROR: Error: Cannot find module '@medic/phone-number'
Require stack:                                                                        
- /home/mrjones/Documents/MedicMobile/cht-core/shared-libs/message-utils/src/index.js
- /home/mrjones/Documents/MedicMobile/cht-core/shared-libs/transitions/src/lib/utils.js
- /home/mrjones/Documents/MedicMobile/cht-core/shared-libs/transitions/src/transitions/index.js
- /home/mrjones/Documents/MedicMobile/cht-core/shared-libs/transitions/src/index.js
- /home/mrjones/Documents/MedicMobile/cht-core/sentinel/src/config.js
- /home/mrjones/Documents/MedicMobile/cht-core/sentinel/server.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:902:15)
    at Function.Module._load (internal/modules/cjs/loader.js:746:27)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:101:18)
    at Object.<anonymous> (/home/mrjones/Documents/MedicMobile/cht-core/shared-libs/message-utils/src/index.js:11:21)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)                                                                                                                       at Function.Module._load (internal/modules/cjs/loader.js:790:12)                                                                                                             at Module.require (internal/modules/cjs/loader.js:974:19) {
  [stack]: "Error: Cannot find module '@medic/phone-number'\n" +   
    'Require stack:\n' +                                                                                                                                                         '- /home/mrjones/Documents/MedicMobile/cht-core/shared-libs/message-utils/src/index.js\n' +                                                                            
    '- /home/mrjones/Documents/MedicMobile/cht-core/shared-libs/transitions/src/lib/utils.js\n' +
    '- /home/mrjones/Documents/MedicMobile/cht-core/shared-libs/transitions/src/transitions/index.js\n' +
    '- /home/mrjones/Documents/MedicMobile/cht-core/shared-libs/transitions/src/index.js\n' +                                                                                    '- /home/mrjones/Documents/MedicMobile/cht-core/sentinel/src/config.js\n' +                                                                                                  '- /home/mrjones/Documents/MedicMobile/cht-core/sentinel/server.js\n' +
    '    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:902:15)\n' +                                                                                        '    at Function.Module._load (internal/modules/cjs/loader.js:746:27)\n' +                                                                                                   '    at Module.require (internal/modules/cjs/loader.js:974:19)\n' +
    '    at require (internal/modules/cjs/helpers.js:101:18)\n' +                                                                                                                '    at Object.<anonymous> (/home/mrjones/Documents/MedicMobile/cht-core/shared-libs/message-utils/src/index.js:11:21)\n' +                                                  '    at Module._compile (internal/modules/cjs/loader.js:1085:14)\n' +
    '    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)\n' +                                                                                          '    at Module.load (internal/modules/cjs/loader.js:950:32)\n' +                                                                                                             '    at Function.Module._load (internal/modules/cjs/loader.js:790:12)\n' +
    '    at Module.require (internal/modules/cjs/loader.js:974:19)',
  [message]: "Cannot find module '@medic/phone-number'\n" +                                                                                                                      'Require stack:\n' +                                                                                                                                                         '- /home/mrjones/Documents/MedicMobile/cht-core/shared-libs/message-utils/src/index.js\n' +
    '- /home/mrjones/Documents/MedicMobile/cht-core/shared-libs/transitions/src/lib/utils.js\n' +
    '- /home/mrjones/Documents/MedicMobile/cht-core/shared-libs/transitions/src/transitions/index.js\n' +
    '- /home/mrjones/Documents/MedicMobile/cht-core/shared-libs/transitions/src/index.js\n' +
    '- /home/mrjones/Documents/MedicMobile/cht-core/sentinel/src/config.js\n' +
    '- /home/mrjones/Documents/MedicMobile/cht-core/sentinel/server.js',
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/mrjones/Documents/MedicMobile/cht-core/shared-libs/message-utils/src/index.js',
    '/home/mrjones/Documents/MedicMobile/cht-core/shared-libs/transitions/src/lib/utils.js',
    '/home/mrjones/Documents/MedicMobile/cht-core/shared-libs/transitions/src/transitions/index.js',
    '/home/mrjones/Documents/MedicMobile/cht-core/shared-libs/transitions/src/index.js',
    '/home/mrjones/Documents/MedicMobile/cht-core/sentinel/src/config.js',
    '/home/mrjones/Documents/MedicMobile/cht-core/sentinel/server.js',
    [length]: 6
  ]
} 
2022-03-01 16:54:12 ERROR: Fatal error intialising medic-sentinel 
2022-03-01 16:54:12 ERROR: Error: Error loading configuration
    at /home/mrjones/Documents/MedicMobile/cht-core/sentinel/src/config.js:75:13
    at processTicksAndRejections (internal/process/task_queues.js:95:5) {
  [stack]: 'Error: Error loading configuration\n' +
    '    at /home/mrjones/Documents/MedicMobile/cht-core/sentinel/src/config.js:75:13\n' +
    '    at processTicksAndRejections (internal/process/task_queues.js:95:5)',
  [message]: 'Error loading configuration'
} 
```

Reply:
I tried removing the extra, I was able to successfully do `npm ci` in lib directory and in the project directory (/cht-core)
Also I quickly started api and sentinel `grunt dev-api` and `grunt dev-sentinel` but didn't experience errors 🤔

Reply:
Oh sorry, I just noticed, so yeah it won't work by running `node server.js`, no even in master.
When we use grunt to start sentinel and api, it will make available the shared-libraries in a place where sentinel and api can find them.
<img width="1067" alt="Screen Shot 2022-03-02 at 6 10 33 pm" src="https://user-images.githubusercontent.com/66472237/156350717-3fd2dd03-1d88-444e-9a1f-f87f8a2d6d98.png">

I think you can remove that extra part from the package-lock.json, it looks okay.

Reply:
oh - you're right! I was just too aggressive in how I reverted it in [2ceb94c](https://github.com/medic/cht-core/pull/7539/commits/2ceb94c677bb5476837531e05c5a228bbec51341) - I pulled the dependency from `package.json` - which was of course wrong!

I've re-removed the change in 1a5d5c902  and I think were there! 

Thanks for digging into this and pushing back to get it clean.  Resolving!